src/frontend/Player/videojs/qualitySelectorPlugin/components/QualitySelectorMenuButton.ts

```diff
@@ -0,0 +1,36 @@
+import videojs from 'video.js';
+
+import { QualitySelectorMenuItem } from './QualitySelectorMenuItem';
+
+const MenuButton = videojs.getComponent('MenuButton');
+
+export class QualitySelectorMenuButton extends MenuButton {
+  constructor(player: videojs.Player, options?: any) {
```

Comment:
Instead of `any`, you could use `videojs.MenuButtonOptions` or create an interface which extends `videojs.MenuButtonOptions` if needed.